rhel/SPECS/alertmanager.spec

```diff
@@ -0,0 +1,64 @@
+%define debug_package %{nil}
+
+%global provider        github
+%global provider_tld    com
+%global project         prometheus
+%global repo            alertmanager
+%global provider_prefix %{provider}.%{provider_tld}/%{project}/%{repo}
+%global import_path     %{provider_prefix}
+%global commit          f74be0400a6243d10bb53812d6fa408ad71ff32d
+%global shortcommit     %(c=%{commit}; echo ${c:0:7})
+
+%global install_golang 0
+
+Name:           percona-%{repo}
+Version:        0.20.0
+Release:        1%{?dist}
+Summary:        The Prometheus monitoring system and time series database
+License:        ASL 2.0
+URL:            https://%{provider_prefix}
+Source0:        https://%{provider_prefix}/archive/%{commit}/%{repo}-%{shortcommit}.tar.gz
+
+%if %{install_golang}
+BuildRequires:   golang >= 1.12.0
+%endif
+
+%description
+%{summary}
+
+%description
+%{summary}
+
+
+%prep
+%setup -q -n %{repo}-%{commit}
+mkdir -p ./build/src/github.com/prometheus
+ln -s $(pwd) ./build/src/github.com/prometheus/alertmanager
```

Comment:
This is wrong and we should not do that in new code: https://jira.percona.com/browse/PMM-5452